flink-metrics/flink-metrics-statsd/src/test/java/org/apache/flink/metrics/statsd/StatsDReporterTest.java

```diff
@@ -81,19 +79,19 @@ public void testAddingMetrics() {
 
         Map<Counter, String> counters = reporter.getCounters();
 
-        assertTrue(counters.containsKey(myCounter));
+        assertThat(counters.containsKey(myCounter)).isTrue();
```

Comment:
```suggestion
        assertThat(counters).containsKey(myCounter);
```
Nit-picking: this is more idiomatic.